tests/test_splicing.py

```diff
@@ -0,0 +1,45 @@
+from fixtures import *  # noqa: F401,F403
+from fixtures import TEST_NETWORK
+from pyln.client import RpcError, Millisatoshi
+from utils import (
+    only_one, wait_for, sync_blockheight, first_channel_id, calc_lease_fee
+)
+
+import pytest
+import re
+import unittest
+import time
+
+@pytest.mark.openchannel('v2')
+def test_splice(node_factory, bitcoind):
+    l1 = node_factory.get_node()
+    l2 = node_factory.get_node()
+    
+    l1.rpc.connect(l2.rpc.getinfo()['id'], 'localhost:%d' % l2.port)
+    l1.openchannel(l2, 4000000)
+
+    result = l1.rpc.splice_init(l2.rpc.getinfo()['id'])
+
+    funds_result = l1.rpc.fundpsbt("100000sat", "slow", 166)
+
+    result = bitcoind.rpc.joinpsbts([result['psbt'], funds_result['psbt']])
+    result = l1.rpc.splice_update(l2.rpc.getinfo()['id'], result)
+    result = l1.rpc.splice_finalize(l2.rpc.getinfo()['id'])
+    result = l1.rpc.signpsbt(result['psbt'])
+    result = l1.rpc.splice_signed(l2.rpc.getinfo()['id'], result['signed_psbt'])
+
+    inv = l2.rpc.invoice(10**2, '1', 'no_1')
+    l1.rpc.pay(inv['bolt11'])
+
+    bitcoind.generate_block(6, wait_for_mempool=1)
```

Comment:
if you wanna be fancy check that the txid for the splice_signed is in the mempool?